semanticcpg/src/test/scala/io/shiftleft/semanticcpg/language/StepsTest.scala

```diff
@@ -9,11 +9,11 @@ import org.json4s._
 import org.json4s.native.JsonMethods.parse
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AnyWordSpec
-import overflowdb.traversal.{Traversal, jIteratortoTraversal, toElementTraversal, toNodeTraversal}
+import overflowdb.traversal.Traversal
 
 class StepsTest extends AnyWordSpec with Matchers {
 
-  val cpg = MockCpg()
+  private val cpg: Cpg = MockCpg()
```

Comment:
```suggestion
  private val cpg = MockCpg()
```
scalac probably asked for a type anntation here, but that's mostly for public members...

Reply:
And it results in faster compilation times.

Reply:
true that